app/services/zendesk_intake_service.rb

```diff
@@ -30,10 +30,7 @@ def assign_requester
       return requester_id
     end
 
-    # failing all else, that's likely noteworthy
-    trace_error('ZendeskIntakeTicketJob failed to create a ticket requester',
-                intake_context(@intake))
-    return # ensure a nil return value
+    raise ZendeskServiceError.new("ZendeskIntakeTicketJob failed to create a ticket requester")
```

Comment:
just want to make sure i understand something - `trace_error` here included an `intake_context` but it looks like every place this method is called was wrapped in a `with_raven_context` so were those two things redundant?